README.md

```diff
@@ -22,37 +22,22 @@ access key and many other aspects of a system. Later on cloud-init will
 also parse and process any optional user or vendor data that was passed to the
 instance.
 
-## Getting help
-
-If you need support, start with the [user documentation](https://docs.cloud-init.io/en/latest/).
-
-If you need additional help consider reaching out with one of the following options:
-
-- Ask a question in the [``#cloud-init`` channel on Matrix](https://matrix.to/#/#cloud-init:ubuntu.com)
-- Follow announcements or ask a question on [GitHub Discussions](https://github.com/canonical/cloud-init/discussions)
-- Find a bug? [Report bugs on GitHub Issues](https://github.com/canonical/cloud-init/issues)
-
-## Distribution and cloud support
-
 The majority of [clouds](https://docs.cloud-init.io/en/latest/reference/datasources.html#datasources_supported)
 and [Linux / Unix OSes](https://docs.cloud-init.io/en/latest/reference/distros.html)
 are supported by and ship with cloud-init. If your distribution or cloud is not
 supported, please get in contact with that distribution and send them our way!
```

Comment:
It doesn't really make sense to have these links to refrence data be stored between "Getting help" and "Start developing", which are both actions that one might take. It makes more sense combined up into the section above which describes cloud-init as a project.